packages/compendium-ui/src/lib/App.tsx

```diff
@@ -1,24 +1,23 @@
-import { Layout } from "antd";
 import "./App.css";
-import { IndexPage } from "./pages/Index";
-import { Settings } from "./components/settings";
-import { SettingsContext } from "./contexts/settings";
-import { useState } from "react";
+import { Route, Routes } from "react-router-dom";
+import RootLayout from "./layouts/root/root";
+import Homepage from "./pages/homepage/homepage";
+import GraphPage from "./pages/graph/graph";
+import NodeType from "./pages/node-type/node-type";
+import ScrollToTop from "./components/scroll-to-top/scroll-to-top";
 
 function App() {
-  const [settings, setSettings] = useState({
-    clusteringEnabled: true,
-  });
-
   return (
-    <SettingsContext.Provider value={{ settings, setSettings }}>
-      <Settings />
-      <Layout style={{ height: "100%" }}>
-        <Layout.Content>
-          <IndexPage />
-        </Layout.Content>
-      </Layout>
-    </SettingsContext.Provider>
+    <>
+      <ScrollToTop />
+      <Routes>
+        <Route path="/" element={<RootLayout />}>
```

Comment:
@zatkoa is this relative or absolute routing? if this app is accessible via aws api gw - we are not able to get a rid of /api/ in url.